src/molecules/select/Select.scss

```diff
@@ -1,50 +1,72 @@
 @import '../../../assets/styles/_globals.scss';
 
```

Comment:
Imo we can add `__font-size` var with design/app friendly font-size, now inherited styles looks like this `font-size: 100%;` and it does not look really good.